### PR TITLE
Make extension's visibility always be on top of any site's headers.

### LINKED ISSUE #172: Some sites have headers that hide the extension
https://www.theverge.com/21346343/gpt-3-explainer-openai-examples-errors-agi-potential

### PR DESCRIPTION
**Pull request by sjonany**
Tested now works on these sites:
- https://www.theverge.com/21346343/gpt-3-explainer-openai-examples-errors-agi-potential
- https://www.technologyreview.com/2020/07/20/1005454/openai-machine-learning-language-generator-gpt-3-nlp/

### CODE REVIEW

File: extension/content.css [hunk 1]
```diff
@@ -2,6 +2,16 @@ readerease-sentence {
 	background-color: inherit !important;
 }
 
+/* Make these containers be on top of any other element, because some websites
+have headers with high z-indices.
+It's okay to be obnoxiously high because user can always deactivate the extension.
+For where the magic value came from, see
+https://stackoverflow.com/questions/491052/minimum-and-maximum-value-of-z-index/25461690#25461690
+ */
+#readingDisplayContainer, #optionsButton, #uiContainer {
```
Comment (sjonany) [inline extension/content.css:9]: no variables for css :( https://stackoverflow.com/a/47508/4143394

**Comment by sjonany** [comment]
One site has z-index: 1100;
The other has z-index: 50000

we just barbarically one-up all of them >: )

File: extension/content.css (hunk 1 above)
Comment (benfrancis314) [reply extension/content.css:9]: Hahahaha

Comment (benfrancis314) [reply extension/content.css:9]: (on the very high z index value; I like it)

**Review by benfrancis314 — APPROVED** [review]
LGTM

File: extension/content.css [hunk 2]
```diff
@@ -2,6 +2,16 @@ readerease-sentence {
...
+	z-index: 16777271;
```
Comment (benfrancis314) [inline extension/content.css:15]: Perhaps we can add a /* Max z-index for Chrome */ comment after this, inline

Comment (sjonany) [reply extension/content.css:15]: Added.

**Pull request merged**